modulo3/projeto-astromatch/astromatch/src/components/ClearMatch.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import axios from "axios";
+
+export default function ClearMatch () {
+    const clearAll = () => {
```

Comment:
Faltou chamar esta função dentro do useEffect para atualizar a tela depois que ela for utilizada